app/models/concerns/cms/addon/sns_poster.rb

```diff
@@ -84,87 +127,87 @@ def connect_twitter
-            open(file.path, 'rb') do |f|
-              image_param << client.upload(f)
-            end
-            i += 1
-            break if i >= 4
+    def post_to_twitter
+      tweet = "#{name}｜#{full_url}"
+      client = connect_twitter
+      image_param = []
+      # 画像の添付があればuploadとupdateを用いて投稿
+      if file_ids.present?
+        i = 0
+        files.each do |file|
+          open(file.path, 'rb') do |f|
+            image_param << client.upload(f)
           end
-          twitter_param = client.update( tweet, { "media_ids"=> image_param.join(',') } )
-          # 画像の添付がなければupdateを用いて投稿
-        else
-          twitter_param = client.update(tweet)
+          i += 1
+          break if i >= 4
         end
-        # 戻り値から投稿IDを取得し、DBに保存
-        twitter_id = twitter_param.id
-        # URLを表示するためにスクリーンネームを取得し、DBに保存
-        user_screen_id = client.user.screen_name
-        self.set(twitter_post_id: twitter_id, twitter_user_id: user_screen_id)
-      rescue => e
-        Rails.logger.fatal("post_to_twitter failed: #{e.backtrace.join("\n  ")}")
+        twitter_param = client.update( tweet, { "media_ids"=> image_param.join(',') } )
+        # 画像の添付がなければupdateを用いて投稿
+      else
+        twitter_param = client.update(tweet)
       end
+      # 戻り値から投稿IDを取得し、DBに保存
+      twitter_id = twitter_param.id
+      # URLを表示するためにスクリーンネームを取得し、DBに保存
+      user_screen_id = client.user.screen_name
+      self.set(twitter_post_id: twitter_id, twitter_user_id: user_screen_id)
+      self.add_to_set(twitter_posted: { twitter_post_id: twitter_id, twitter_user_id: user_screen_id })
+    rescue => e
+      Rails.logger.fatal("post_to_twitter failed: #{e.backtrace.join("\n  ")}")
+    end
 
-      def post_to_facebook
-        message = message_format(html)
-        if file_ids.present?
-          image_path = files.first.full_url
-        end
-        access_token = self.site.facebook_access_token
-        graph = Koala::Facebook::API.new(access_token)
-        # facebokに投稿し、戻り値を取得
-        facebook_params = graph.put_wall_post(
+    def post_to_facebook
+      message = message_format(html)
+      file_ids.present? ? image_path = files.first.full_url : nil
+      access_token = self.site.facebook_access_token
+      graph = Koala::Facebook::API.new(access_token)
+      # facebokに投稿し、戻り値を取得
+      facebook_params = graph.put_wall_post(
           message, {
-            "name"=> "#{name} - #{site.name}",
-            "link"=> full_url,
-            "picture"=> image_path,
-            "description"=> description
-          }
-        )
-        facebook_param = facebook_params['id'].to_s
-        # 戻り値からUID/PID取得し、DBに保存
-        facebook_id_array = facebook_id_separator(facebook_param)
-        self.set(facebook_user_id: facebook_id_array[0], facebook_post_id: facebook_id_array[1])
-      rescue => e
-        Rails.logger.fatal("post_to_facebook failed: #{e.backtrace.join("\n  ")}")
-      end
+          "name"=> "#{name} - #{site.name}",
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->